tests/installation/addon_products_sle.pm

```diff
@@ -126,6 +127,21 @@ sub handle_addon {
     }
 }
 
+sub test_addonurl {
+    my $testvalue = get_var('ADDONURL');
+    my @missing_modules;
+    my @test_modules = split(/,/, get_var('WORKAROUND_MODULES'));
```

Comment:
Ups, I have forgotten to replace this one with the list of modules. I was just lazy to look after all of the possible modules, while I was writing the code. Thanks!

Reply:
Actually I recall why I have chosen this solution. https://openqa.suse.de/tests/1824434/file/vars.json WORKAROUND_MODULES is basically hardcoded in openQA test definition, therefore always available and we are selecting only a portion of all possible modules.

Reply:
I am trying to access it on L#193

Reply:
Sure, I saw that, but as I've said we could do quick check here as we validate settings before trying to do anything about it. So in case of unavailable module we will fail earlier and hence save some computing time.
Later on we can also wait when YaST will fail, as it won't be able to access the link and will give transparent error message. WDYT?

Reply:
That make sense, so in case we have the variable set properly, test will die immediately if the repo is not accessible.